models/marts/ledger_current_state/contract_code_current.yml

```diff
@@ -3,7 +3,15 @@ version: 2
 models:
   - name: contract_code_current
     description: '{{ doc("contract_code_current") }}'
-
+    tests:
+      - dbt_utils.recency:
+          datepart: hour
+          field: closed_at
+          interval: 12
```

Comment:
Have you profiled the data to know if this interval window is large enough to not trigger false positives? `contract_code` is another table that is updated infrequently. Might be worthwhile to profile occurrence of writes to this table.

Reply:
Or update the severity to warn

Reply:
You are right, it does not change frequently. Will increase the thresholds
<img width="1275" alt="Screenshot 2024-09-19 at 8 34 08 PM" src="https://github.com/user-attachments/assets/b80d3a02-f307-48cd-92ab-950ee386e956">